src/lib/components/references/bibleTextReferenceFetcher.ts

```diff
@@ -280,17 +180,41 @@ function getMultiVersePassageContentFromBookTexts(
     return passageContent;
 }
 
-function generateVerseReference(bookTexts: BibleBookTexts[], language: Language, hasDifferentMapping: boolean): string {
+function generateVerseReference(
+    sourceBibleStartVerseId: number,
+    sourceBibleEndVerseId: number,
+    targetBibleTextRange: BibleBookTexts[],
+    language: Language,
+    hasDifferentMapping: boolean
+): string {
     // if both 0 and verse 1 are requested we still only want to display a single reference to only verse 1
     if (
-        bookTexts.length === 1 &&
-        bookTexts[0]!.chapters.length === 1 &&
-        bookTexts[0]!.chapters[0]!.verses.filter((v) => v.number !== 0).length === 1
+        targetBibleTextRange.length === 1 &&
+        targetBibleTextRange[0]!.chapters.length === 1 &&
+        targetBibleTextRange[0]!.chapters[0]!.verses.filter((v) => v.number !== 0).length === 1
     ) {
-        const singleBookText = bookTexts[0]!;
+        const singleBookText = targetBibleTextRange[0]!;
         const singleChapter = singleBookText.chapters[0]!;
         const singleVerse = singleChapter.verses.at(-1)!;
 
+        // there's one mapping that we don't want to display as different: where the source verse range is 0-1 and the target verse is 1
+        if (hasDifferentMapping) {
```

Comment:
This is the only new logic.  We were previously displaying, for example, `Psalms 11:1*` when mapping from `Psalms 11:0-1` which doesn't make sense to the user.  Those shouldn't have the `*`.